packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFURLTests.m

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Flutter;
+@import XCTest;
+@import webview_flutter_wkwebview;
+
+#import <OCMock/OCMock.h>
+
+@interface FWFURLTests : XCTestCase
+@end
+
+@implementation FWFURLTests
+- (void)testAbsoluteString {
+  NSURL *mockUrl = OCMClassMock([NSURL class]);
+  OCMStub([mockUrl absoluteString]).andReturn(@"https://www.google.com");
+
+  FWFInstanceManager *instanceManager = [[FWFInstanceManager alloc] init];
+  [instanceManager addDartCreatedInstance:mockUrl withIdentifier:0];
```

Comment:
Wasn't there a recent change that required this to be non-zero, or am I confusing this with something else?

Reply:
`addDartCreatedInstance` should be able to take a 0 since the Dart side reserves all identifiers 0->2^16